packages/web-components/src/accordion/accordion.styles.ts

```diff
@@ -1,16 +1,23 @@
 import { css } from '@microsoft/fast-element';
 import { display } from '@microsoft/fast-foundation';
-import { accentFillRestBehavior, neutralDividerRestBehavior, neutralForegroundRestBehavior } from '../styles/';
+import {
+  bodyFont,
+  neutralDivider,
+  neutralForegroundRest,
+  outlineWidth,
+  typeRampMinus1FontSize,
+  typeRampMinus1LineHeight,
+} from '../design-tokens';
 
 export const accordionStyles = (context, definition) =>
   css`
     ${display('flex')} :host {
       box-sizing: border-box;
       flex-direction: column;
-      font-family: var(--body-font);
-      font-size: var(--type-ramp-minus-1-font-size);
-      line-height: var(--type-ramp-minus-1-line-height);
-      color: ${neutralForegroundRestBehavior.var};
-      border-top: calc(var(--outline-width) * 1px) solid ${neutralDividerRestBehavior.var};
+      font-family: ${bodyFont};
+      font-size: ${typeRampMinus1FontSize};
+      line-height: ${typeRampMinus1LineHeight};
+      color: ${neutralForegroundRest};
+      border-top: calc(var(--outline-width) * 1px) solid ${neutralDivider};
```

Comment:
I think outline-width should exist as a token, yea?

Reply:
Yeah, good catch - there may be a few missed. I did find and replace through most all files individually but it's possible some were missed. Will appreciate the close look here.